src/FreeScribe.client/UI/MainWindowUI.py

```diff
@@ -67,6 +68,9 @@ def create_docker_status_bar(self):
         Create a Docker status bar to display the status of the LLM and Whisper containers.
         Adds start and stop buttons for each container.
         """
+        # if not feature do not do this
```

Comment:
We kind of need the duplicate checks in our functions because the ui can be dynamically updated with the remove and create functions therefore we need to preform checks before either destroying/creating not just on application start...